### ISSUE TITLE
EGDI WMS and WFS services do not have service metadata entries

### ISSUE THREAD

**Issue by nmtoken** [GeoEra-GIP/Project-Support-WP8#528]
The backbone services for the EGDI platform (currently [WMS](https://data.geus.dk/egdi/wms/?SERVICE=WMS&REQUEST=GetCapabilities) and [WFS](https://data.geus.dk/egdi/wms/?SERVICE=WFS&REQUEST=GetCapabilities)) do not appear to have service metadata records in the [EGDI Metadata catalogue](http://www.europe-geology.eu/metadata/).

Considering [FAIR principles](https://www.go-fair.org/fair-principles/)  should we have such metadata to improve Findability and Accessibility to the data (and representations of the data)?

**Comment by Kramolisova** [comment]
You're absolutely right. We have also pointed this out several times, these are some services that were not created within GeoERA, but before.
Bjarni or Martine,@mh-geus-dk, can you use a query to find out which services are missing metadata. We (EGDI partners) should describe them or pass on a request to add metadata to the people concerned.

**Comment by nmtoken** [comment]
Metadata is required for 

[WMS](https://data.geus.dk/egdi/wms/?service=WMS&request=GetCapabilities)  and

[WFS](https://data.geus.dk/egdi/wms/?service=WFS&request=GetCapabilities)
